wallet/src/events.rs

```diff
@@ -46,9 +48,8 @@ impl ManagedList {
             id: events.len() as u32,
             timestamp,
         });
-        if events.len() > 1000 {
-            events.drain(..events.len() - 1000);
-            canister.culled = Some(events[0].id as usize);
+        if events.len() > MAX_CANISTER_EVENTS {
+            events.drain(..events.len() - MAX_EVENTS);
```

Comment:
shouldn't these both be using one of the other of MAX_EVENTS or MAX_CANISTER_EVENTS?